package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "bitburner",
```

Comment:
I've been avoiding checking in changes to this file. Maybe it should be gitignored?

Reply:
I never changed these files manually by using npm run doc think you automatically changed files.